tss-esapi/src/context/tpm_commands/object_commands.rs

```diff
@@ -154,7 +149,7 @@ impl Context {
                 self.optional_session_2(),
                 self.optional_session_3(),
                 private,
-                public,
+                &(*public).clone().into(),
```

Comment:
That looks a bit weird, is the `$(*...)` part needed? Would've thought a simple `public.clone()` was enough

Reply:
I had some problems with this. It created a clone of a reference to a public which was not what I wanted.

Reply:
I fixed it now.